platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/snapshotter/MapSnapshotter.java

```diff
@@ -283,7 +296,7 @@ public void cancel() {
    *
    * @param mapSnapshot the map snapshot to draw the overlay on
    */
-  protected void addOverlay(MapSnapshot mapSnapshot) {
+  private void addOverlay(MapSnapshot mapSnapshot) {
```

Comment:
End developers use this method to draw image overlays between the map and our logo / attribution. Eg. a user wants to show a weather map on top of a map image. To make this work with logo / attribution, they will have to override this method, draw the overlay and call into super.addOverlay to add the logo/attribution. Would prefer keeping this protected to support those cases.

Reply:
Oops, I thought I knew that! Not sure why I made this mistake...